highcharts-shared-styles.html

```diff
@@ -1,10 +1,11 @@
+<link rel="import" href="../shadycss/apply-shim.html">
```

Comment:
Is this required?

Reply:
Yes it is. [here](https://www.polymer-project.org/2.0/docs/upgrade#class-based-elements-import-the-css-mixin-shim)